### PR TITLE
Small tweaks

### PR DESCRIPTION
**Pull request by alicebartlett**
Haven't caught up with design on this yet, but here are a few changes for you to look at.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +23,19 @@ Use the following HTML to get a full-width FT affiliation ribbon.  Ideally this
...
-		</div>	
+		</div>
 	</div>
 </div>
 ```
 
 ### Sass
 
-As with all Origami components, o-ft-affiliate-ribbon has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) set `$o-o-ft-affiliate-ribbon-is-silent : false;` in your Sass after you've imported the o-ft-affiliate-ribbon Sass.
+As with all Origami components, o-ft-affiliate-ribbon has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) set `$o-o-ft-affiliate-ribbon-is-silent : false;` in your Sass after you've imported the o-ft-affiliate-ribbon Sass:
```
Comment (onishiweb) [inline README.md]: "after" should be "before" in the line `in your Sass after you've imported`

Also, the variable has a double-o prefix `$o-o-`

**Review by onishiweb — CHANGES_REQUESTED** [review]

**Comment by alexwilson** [comment]
Sorry, just introduced a conflict by merging another another PR resolving the double-prefix typo!

The other changes look good, didn't know that you could version imagesets!

**Review by onishiweb — APPROVED** [review]

**Comment by alexwilson** [comment]
I wasn't aware of situation with typography...  That, and the background-image mixins duplicated between different components have made me wonder whether it might be worth introducing utility Sass mixins for inclusion between components.

**Review by alexwilson — APPROVED** [review]

**Comment by alexwilson** [comment]
Cheers all!

If there's any further changes to fonts etc, we can review later!

**Pull request merged**